lms/djangoapps/teams/tests/test_views.py

```diff
@@ -597,6 +598,10 @@ def login(self, user):
             student_inactive.is_active = False
             student_inactive.save()
         else:
+            if user not in self.users:
```

Comment:
Still not sure the root cause but apparently it happend due to `setuptestdata` so just fix it.